style.css

```diff
@@ -0,0 +1,999 @@
+
+.contact-me {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  text-align: center;
+  font-weight: 700;
+  font-size: 40px;
+  padding: 23px;
+  color: hsl(0, 0%, 100%);
+}
+
+p.Content {
+  padding-left: 1px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  font-size: 17px;
+  color: #344563;
+}
+
+.about-me {
+  font-weight: 700;
+  font-family: "Poppins", sans-serif;
+  font-size: 50px;
+  margin: 2px;
+  align-items: center;
+  color: hsl(218, 54%, 20%);
+}
+
+.skill-title {
+  margin-bottom: 10px;
+  padding-bottom: 20px;
+}
+
+.about_myself__menu {
+  width: 100%;
+  padding-top: 50px;
+}
+
+.Skill-border {
+  border-bottom: 1px solid #c8d1e776;
+}
+
+/* popup windows design   */
+.Project-container {
+  margin: auto 3vw;
+}
+
+.poup-container {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  margin: 0 auto;
+  display: flex;
+  align-items: flex-start;
+  justify-content: center;
+  overflow: hidden;
+}
+
+.works {
+  background-color: #f7f7f7;
+  padding: 12.8rem 0;
+  padding-bottom: 12.8rem;
+  margin-bottom: 100px;
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 12.8rem;
+}
+
+.project-card {
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 1.6rem;
+  padding: 10px;
+  margin-bottom: 12rem;
+  display: flex;
+  flex-direction: column;
+  gap: 1.2rem;
+  z-index: 9;
+}
+
+.popup {
+  position: fixed;
+  width: 100%;
+  background-color: rgba(193, 199, 208, 0.7);
+  backdrop-filter: blur(5px);
+  padding-top: 5vh;
+  height: 100vh;
+  top: 0;
+  z-index: 999;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.project-title {
+  display: flex;
+  align-items: center;
+  padding-left: 15px;
+  justify-content: space-between;
+}
+
+.popup-close-btn {
+  background: none;
+  border: none;
+  cursor: pointer;
+}
+
+.popup-close-icon {
+  width: 3.2rem;
+  color: #67798e;
+}
+
+.popup-btn {
+  display: flex;
+  gap: 0.4rem;
+  align-items: center;
+  justify-content: center;
+  font-size: 1.2rem;
+  padding: 0.6rem 1rem;
+}
+
+.check-btn {
+  display: flex;
+  gap: 1rem;
+  font-size: 1.5rem;
+}
+
+.line {
+  display: flex;
+  position: relative;
+  gap: 1rem;
+}
+
+.popup-logo {
+  display: flex;
+  align-items: center;
+}
+
+.popup-btn:hover a .popup-logo,
+.popup-btn:active {
+  fill: #fff;
+}
+
+.project-lang,
+.check-btn {
+  justify-content: center;
+}
+
+.popup-img {
+  margin-bottom: 1rem;
+  max-width: 100%;
+  height: auto;
+}
+
+.project-img {
+  border-radius: 0.8rem;
+  width: 98%;
+  object-fit: contain;
+}
+
+.heading-project {
+  color: #172b4d;
+  font-size: 3.2rem;
+  font-weight: 700;
+  margin-right: 1rem;
+}
+
+.About-project {
+  color: #344563;
+  font-size: 1.5rem;
+  line-height: 2.4rem;
+  padding-right: 10%;
+}
+
+.project-detail {
+  display: flex;
+  flex-direction: column;
+  gap: 2rem;
+  padding: 10px;
+}
+
+.popup-details {
+  flex-direction: row;
+  gap: 0.3rem;
+  padding-top: 2rem;
+}
```

Comment:
- [ ] Please ensure that for mobile screens, the `flex-direction` is set to column so that you achieve the same design with the Figma guidelines
### Your Design
![image](https://github.com/melkamu12/Portfolio-DesktopVersionM/assets/46839548/b5a7cc2c-4170-4ae4-ac7e-c365d22400a4)
### Figma Design
![image](https://github.com/melkamu12/Portfolio-DesktopVersionM/assets/46839548/fc81759b-909b-41f2-b955-ba3b9a49a854)